channels_points.go

```diff
@@ -104,6 +106,36 @@ type DeleteCustomRewardsResponse struct {
 	ResponseCommon
 }
 
+type UpdateChannelCustomRewardsRedemptionStatusParams struct {
+	ID            string `query:"id"`
+	BroadcasterID string `query:"broadcaster_id"`
+	RewardID      string `query:"reward_id"`
+	Status        string `json:"status"`
+}
+
+type ChannelCustomRewardsRedemptionResponse struct {
+	ResponseCommon
+	Data ManyChannelCustomRewardsRedemptions
+}
+
+type ManyChannelCustomRewardsRedemptions struct {
+	Redemptions []ChannelCustomRewardsRedemption `json:"data"`
+}
+
+type ChannelCustomRewardsRedemption struct {
+	ID               string              `json:"id"`
+	BroadcasterID    string              `json:"broadcaster_id"`
+	BroadcasterLogin string              `json:"broadcaster_login"`
+	BroadcasterName  string              `json:"broadcaster_name"`
+	UserID           string              `json:"user_id"`
+	UserName         string              `json:"user_name"`
+	UserLogin        string              `json:"user_login"`
+	UserInput        string              `json:"user_input"`
+	Status           string              `json:"status"`
+	RedeemedAt       time.Time           `json:"redeemed_at"`
```

Comment:
This package has it's own implementation for handling [time](https://github.com/nicklaw5/helix/blob/main/time.go) values. Please make the following adjustment:

```suggestion
	RedeemedAt       Time           `json:"redeemed_at"`
```

Reply:
Updated. Sorry I guess I missed that while I was skimming the package